webui-src/app/mail/mail_resolver.js

```diff
@@ -22,6 +22,7 @@ const Messages = {
   load() {
     rs.rsJsonApiRequest('/rsMsgs/getMessageSummaries', {}, (data) => {
       Messages.all = data.msgList;
+      console.log(Messages.all);
```

Comment:
Would be nice if you could remove these logs since they clutter the console, but not very important.